lib/server.js

```diff
@@ -13,13 +13,23 @@ const debug = require('debug')('tinylr:server');
 const CONTENT_TYPE = 'content-type';
 const FORM_TYPE = 'application/x-www-form-urlencoded';
 
+function wrapStream (path) {
+  return function () {
+    return fs.createReadStream(path);
+  };
+}
+
 class Server extends events.EventEmitter {
   constructor (options = {}) {
     super();
 
     this.options = options;
 
-    options.livereload = options.livereload || require.resolve('livereload-js/dist/livereload.js');
+    options.livereload = typeof options.livereload === 'string'
```

Comment:
Can you put `? ...` at the end of the line instead of line breaking before `?` ?

```js
options.livereload = typeof options.livereload === 'string' ? wrapStream(options.livereload)
  : typeof options.livereload === 'function' ? options.livereload
  : wrapStream(require.resolve('livereload-js/dist/livereload.js'));
```